### PR TITLE
implement image borders

### PR DESCRIPTION
**Pull request by christinaausley**
## What is the purpose of the change

Based on https://github.com/camunda/developer-experience/issues/24, this PR adds a soft border to all images within Markdown content.

### What does this look like?

The custom CSS adds a soft gray border to images, as follows:
Screen Shot 2023-01-24 at 11 09 56 AM
Screen Shot 2023-01-24 at 11 10 18 AM

### What does this apply to and not apply to?

This applies solely to images, including PNG files.

This does not apply to BPMN files.

### Possible pitfalls

As discussed with @pepopowitz, naturally we will want to plan how we can individually turn this border off for specific images in the case unique circumstances require us to.

We workshopped the idea of a React component for a "borderless image" -- if this is worth pursuing, @pepopowitz and I can work on this supplementally.

### Questions to answer

1. @pepopowitz This is where I could use your help. The two of us discussed not including borders for images within tables, but I can't quite get this to work. Am I missing something? @akeller How do you feel about excluding borders in images within tables? Do you have a particular opinion on this? Here is how it currently looks:
Screen Shot 2023-01-24 at 11 10 32 AM

2. I've skimmed through the documentation locally, but do you see any other examples where we would/would not want to include a border?

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -349,3 +349,14 @@ span.callout + p::after {
...
+.theme-doc-markdown img {
+  border-style: solid;
+  border-color: lightgray;
+  border-width: 1px;
+  box-shadow: 4px 4px 4px 4px lightgray;
+}
+
+.theme-doc-markdown table img {
+  border: none;
+}
```
Comment (pepopowitz) [inline src/css/custom.css:362]:
```suggestion
.theme-doc-markdown table img {
  border: none;
  box-shadow: none;
}
```

@christinaausley I think this is what you're looking for to remove the borders inside of tables. In addition to resetting the `border` property, we also need to reset the `box-shadow` property. 

(If you look closely at the images inside of the table, or inspect in your browser tools, you'll see that they *don't* actually have a border property; you reset it properly in this code. They *do* still have a box-shadow property, and that's what you're seeing.)

File: src/css/custom.css [hunk 2]
```diff
@@ -349,3 +349,14 @@ span.callout + p::after {
 .dmn-decision-table-container .tjs-table {
   width: 100%;
 }
+
+.theme-doc-markdown img {
+  border-style: solid;
+  border-color: lightgray;
+  border-width: 1px;
+  box-shadow: 4px 4px 4px 4px lightgray;
```
Comment (pepopowitz) [inline src/css/custom.css:357]:
```suggestion
  box-shadow: 4px 4px 4px 1px lightgray;
```

I'm 100% certain you were working off of my code, but I recommend playing with the last numeric value of this property. I changed it to 1px, for example, and I think it gives a softer edge, and removes the alignment effect from your original value: 

image

**Comment by pepopowitz** [comment]
I also want to tag @ingorichtsmeier directly, as he [originally brought up the concern of putting a border around BPMN images](https://github.com/camunda/developer-experience/issues/24#issuecomment-1380470774). Do the images shared in this PR qualify as "not easily confused with a Pool symbol?

File: src/css/custom.css (hunk 1 above)
Comment (christinaausley) [reply src/css/custom.css:362]: THANK YOU! 🥳

**Comment by akeller** [comment]
It's a bit nitpicky. The top of this image _looks_ weird - 

Model your first process running on localhost

But on the production page, I noticed this image could be trimmed a bit, and this weirdness would likely go away. There is a weird shadow on the image across the middle of the bottom. It's visible in both.

Model your first process running on production

On some pages, there is no gap between the image and the text above it and IMO looks more obvious with the shadows. Is this something that we'll fight prettier formatting on and need to modify with CSS?

**Comment by christinaausley** [comment]
@akeller When you talk about the trimming, do you mean the box shadow I applied here, or the way the actual screenshot looks?

> On some pages, there is no gap between the image and the text above it and IMO looks more obvious with the shadows. Is this something that we'll fight prettier formatting on and need to modify with CSS?

@pepopowitz do you have thoughts on the above?

**Comment by akeller** [comment]
The shadow is in the screenshot itself, just like the top of the image I'm commenting on is part of the screenshot, not your added soft gray border. This is feedback for the screenshot taker, not you 😄 

> @akeller How do you feel about excluding borders in images within tables? Do you have a particular opinion on this?

I realized I didn't come back and provide my full response. Excluding borders in images within tables makes a ton of sense because you don't need additional emphasis in a table. That said, it's not a deal breaker if we are having trouble implementing it. It can always be a future iteration!

**Comment by christinaausley** [comment]
> The shadow is in the screenshot itself, just like the top of the image I'm commenting on is part of the screenshot, not your added soft gray border. This is feedback for the screenshot taker, not you 😄

**I'll do some digging on this to see why it may be applied to all or just a subset of our images.**

> Excluding borders in images within tables makes a ton of sense because you don't need additional emphasis in a table. That said, it's not a deal breaker if we are having trouble implementing it. It can always be a future iteration!

**This has been resolved in a recent commit, so no borders will be applied within tables 👍 **

> On some pages, there is no gap between the image and the text above it and IMO looks more obvious with the shadows. Is this something that we'll fight prettier formatting on and need to modify with CSS?

**This is the only question we still need to answer. @akeller Do you feel this is a blocking comment, or one we can iterate moving forward?**

**Moving this PR out of `draft` and `hold` as we're nearing the finish line.**

**Comment by akeller** [comment]
> This is the only question we still need to answer. @akeller Do you feel this is a blocking comment, or one we can iterate moving forward?

Good to move forward and iterate @christinaausley 🚀

**Review by akeller — APPROVED** [review]

**Review by pepopowitz — APPROVED** [review]

**Pull request merged**